src/ncp/spinel.c

```diff
@@ -1873,6 +1877,28 @@ const char *spinel_net_role_to_cstr(uint8_t net_role)
     return ret;
 }
 
+const char *spinel_mcu_power_state_to_cstr(spinel_mcu_power_state_t mcu_power_state)
+{
+    const char *ret = "MCU_POWER_STATE_UNKNONW";
```

Comment:
typo: `MCU_POWER_STATE_UNKNONW` -> `MCU_POWER_STATE_UNKNOWN`

Reply:
Thanks... fixed it. :)